Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
-    minitest (5.14.3)
+    mini_mime (1.0.3)
+    minitest (5.14.4)
     msgpack (1.4.2)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
     net-ssh (6.1.0)
-    nio4r (2.5.5)
-    nokogiri (1.11.1-x86_64-darwin)
+    nio4r (2.5.7)
+    nokogiri (1.11.3-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
-    pagy (3.10.0)
+    pagy (3.11.0)
     parallel (1.20.1)
-    parser (3.0.0.0)
+    parser (3.0.1.0)
       ast (~> 2.4.1)
     pg (1.2.3)
-    popper_js (1.16.0)
+    popper_js (2.8.6)
     public_suffix (4.0.6)
-    puma (5.2.1)
+    puma (5.2.2)
       nio4r (~> 2.0)
     racc (1.5.2)
     rack (2.2.3)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (6.1.1)
-      actioncable (= 6.1.1)
-      actionmailbox (= 6.1.1)
-      actionmailer (= 6.1.1)
-      actionpack (= 6.1.1)
-      actiontext (= 6.1.1)
-      actionview (= 6.1.1)
-      activejob (= 6.1.1)
-      activemodel (= 6.1.1)
-      activerecord (= 6.1.1)
-      activestorage (= 6.1.1)
-      activesupport (= 6.1.1)
+    rails (6.1.3.1)
```

Comment:
### [rails](https://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.1.1 → 6.1.3.1

[_[change-log](https://github.com/rails/rails/releases/tag/v6.1.3.1), [source-code](https://github.com/rails/rails/tree/v6.1.3.1)_]





<details>
<summary>Commits</summary>

A change of **80** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.1.1...v6.1.3.1).

These are the first 10 commits:
- (689d096) [Merge pull request #41049 from rails/revert-39321-fix_update…](https://github.com/rails/rails/commit/689d0963e0e79f5daf0d3962c8da410b501ebaec)
- (5284a16) [Merge pull request #41030 from eileencodes/fix-preventing-wr…](https://github.com/rails/rails/commit/5284a165b8e8821fcfca62dac360bd9ed7a674b4)
- (67fd5cb) [Fix failing tests introduced by #41030](https://github.com/rails/rails/commit/67fd5cb3efe6cbd31ad88261abe9a03f186811c5)
- (9f68683) [Merge pull request #41050 from alea12/update-actioncable-doc…](https://github.com/rails/rails/commit/9f68683e57adbe7aad544e3c1c2c9e264517ca66)
- (044236e) [Merge pull request #41052 from bogdanvlviv/preload_links_hea…](https://github.com/rails/rails/commit/044236e3beec448d78bdcc3dd5209c6e90a8fc50)
- (90339be) [Merge pull request #41063 from kamipo/fix_where_with_polymor…](https://github.com/rails/rails/commit/90339be70c770e2f975e099c80f0b4462ad63297)
- (9cc99d7) [Merge pull request #40704 from alpaca-tc/sandbox_multiple_da…](https://github.com/rails/rails/commit/9cc99d70c991b586ec9e74fbc391b599eba37bb8)
- (12bb9d3) [Merge pull request #41082 from kamipo/fix_update_optimistic_…](https://github.com/rails/rails/commit/12bb9d32f56883914abcd98fd72e3c68c444808d)
- (acb6631) [Merge pull request #41090 from kamipo/fix_find_by_with_custo…](https://github.com/rails/rails/commit/acb6631cd99cdfe7db356773ef74cad7cbb570ed)
- (3f4fde4) [Merge pull request #41098 from kamipo/fix_source_type_with_s…](https://github.com/rails/rails/commit/3f4fde4d9f804140be8304b524792c052e3d1024)


</details>